client/scss/components/_header.scss

```diff
@@ -0,0 +1,289 @@
+//* @define header; weak
+
+.header {
+  padding-top: 0.8rem;
+}
+
+.header__upper-wrap {
```

Comment:
Yeah, this was less than ideal – a slightly lazy inclusion of a wrapping element with new styles after the fact, instead of including an inner element and restyling.

Have updated.